project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject com.taoensso/sente "1.5.0"
+(defproject com.taoensso/sente "1.6.0"
```

Comment:
Not sure if this is correct, but it seemed like Break Versioning would imply a bump.